gemfiles/Gemfile.common.rb

```diff
@@ -11,7 +11,7 @@
 group :test do
   gem 'test-unit',      '~> 3.0'
   gem 'rspec',          '~> 3.4.0'
-  gem 'timecop',        '= 0.6.1'
+  gem 'timecop',        '>= 0.9.4'
```

Comment:
Needs timecop 0.9.4+ for ruby 3.1+.
https://github.com/travisjeffery/timecop/pull/288